frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ManagedClusters.tsx

```diff
@@ -749,6 +787,23 @@ export function useClusterDistributionColumn(
         resource={'managedclusterpage'}
       />
     ),
+    exportContent: (cluster) => {
+      let version
+      const openshiftText = 'OpenShift'
+      const microshiftText = 'MicroShift'
+
+      if (cluster?.provider === Provider.microshift) {
+        version = cluster?.microshiftDistribution?.version
+        return version ?? `${microshiftText} ${version}`
+      }
+      // use version from cluster image
+      const clusterImageVersion = agentClusterObject[cluster.name]
+      if (clusterImageVersion) {
+        return `${openshiftText} ${clusterImageVersion}`
+      }
+      // else use displayVersion
+      return cluster?.distribution?.displayVersion
+    },
```

Comment:
Regarding the AI cluster draft version that was missing, I missed two other cases for getting the cluster version: microshift, and the cluster image. I've added them to the callback logic.